Documentation/workflow/DevelopmentTips.md

```diff
@@ -373,3 +373,69 @@ steps rely on having the Android NDK installed.
         (gdb) set sysroot /tmp/gdb-symbols
         (gdb) set solib-search-path /tmp/gdb-symbols
         (gdb) target remote :50999
+
+# .NET 6 Tips
+
+## Finding Mono runtime packs
+
+`main` builds of the Mono runtime packs are on the following NuGet
+feed, such as this `nuget.config`:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
+Internal MS users can view packages on Azure DevOps here:
```

Comment:
This URL is public, in that I can access it from an Incognito window without logging in.

Reply:
and we should write out "Microsoft", not use "MS". :-)

Reply:
I didn't realize that site is public -- so took that out.